src/components/Modal/Modal.js

```diff
@@ -305,7 +305,7 @@ export default class Modal extends Component {
         className={`${prefix}--modal-close`}
         type="button"
         onClick={onRequestClose}
-        title={iconDescription}
```

Comment:
Seems that we lost hover-over text on the close button. Probably we'd keep both?

Reply:
would it be better to keep the `title` on the button and add the `aria-label` attribute to the SVG itself instead of replacing the attribute entirely?